src/Symfony/Component/Workflow/MarkingStore/MarkingStoreInterface.php

```diff
@@ -36,8 +36,7 @@ public function getMarking($subject);
     /**
      * Sets a Marking to a subject.
      *
-     * @param object  $subject A subject
-     * @param Marking $marking A marking
+     * @param object $subject A subject
      */
-    public function setMarking($subject, Marking $marking);
+    public function setMarking($subject, Marking $marking, array $context = []);
```

Comment:
@lyrixx  This is a BC break.

Reply:
Oups. I did not think it's gonna break the BC. I will fix it tomorrow. 

Thanks for the feedback